File: prime.py


def find_n_prime_numbers(n):
    prime_numbers = []
    count = 0
    num = 2
    while count < n:
        if is_prime(num):
            prime_numbers.append(num)
            count += 1
        num += 1
    return prime_numbers

# Find the first 10 prime numbers
print(find_n_prime_numbers(10))

File: prime.js
function isPrime(num) {
  for (let i = 2; i < num; i++) {
    if (num % i === 0) {
      return false;
    }
  }
  return true;
}

function findFirstNPrimes(n) {
  let primeNumbers = [];
  let num = 2;
  while (primeNumbers.length < n) {
    if (isPrime(num)) {
      primeNumbers.push(num);
    }
    num++;
  }
  return primeNumbers;
}

let n = 10;
console.log(`The first ${n} prime numbers are:`, findFirstNPrimes(n));
